src/Caching/Impl/ModuleFactory.cs

```diff
@@ -105,6 +105,10 @@ public IPythonType ConstructType(string qualifiedName)
                     return null;
                 }
 
+                if(nextModel is SuperModel sm) {
+                    sm.Mro = typeArgs;
+                }
```

Comment:
```memberName = "super"```, but I need a way to pass the mro to the superModel, so I made something specific here, then just below Create gets called